src/components/App/FeedbackGif/FeedbackGif.module.scss

```diff
@@ -0,0 +1,23 @@
+@import 'src/styles/abstracts/variables';
+
+.gifEmbed {
+  border-left: 0.6rem solid;
+  border-right: 0.6rem solid;
+  border-radius: 0.2rem;
+  height: auto;
+  width: auto;
+
+  div {
+    img {
```

Comment:
is this img the same as Image of next/image

Reply:
yep